pandas/tests/frame/test_api.py

```diff
@@ -243,9 +243,9 @@ def test_itertuples(self):
     def test_len(self):
         assert len(self.frame) == len(self.frame.index)
 
-    def test_as_matrix(self):
+    def test_values(self):
```

Comment:
Can you leave one test (or now add a `test_as_matrix_deprecated` that copies eg the first case of this test) that uses `as_matrix` and that asserts it raises a warning and is the same as `.values`

Reply:
Ok. I've added in below ``test_repr_with_mi_nat``